app/graphql/types/user_type.rb

```diff
@@ -4,24 +4,28 @@ class UserType < Types::BaseObject
     field :name, String, null: false
     field :title, String, null: false
     field :avatar_url, String, null: true
+    field :name, String, null: false
+    field :preferred_name, String, null: true
+
+    delegate :name, to: :object
 
     def avatar_url
       BatchLoader::GraphQL
         .for(object.id)
         .batch do |user_ids, loader|
-          User
-            .includes(avatar_attachment: :blob)
-            .where(id: user_ids)
-            .each do |user|
-              if user.avatar.attached?
-                url =
-                  Rails.application.routes.url_helpers.rails_public_blob_url(
-                    user.avatar_variant(:thumb)
-                  )
-                loader.call(user.id, url)
-              end
-            end
+        User
+          .includes(avatar_attachment: :blob)
+          .where(id: user_ids)
+          .each do |user|
+          if user.avatar.attached?
+            url =
+              Rails.application.routes.url_helpers.rails_public_blob_url(
+                user.avatar_variant(:thumb)
+              )
+            loader.call(user.id, url)
+          end
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)